### DISCUSSION TITLE
Export from Unity to Blender to render

### DISCUSSION THREAD

**Discussion by akutterxressential** [atteneder/glTFast#469] [Q&A] [answered]
Hello everyone,

I am wondering if it is possible to export a GLB file (and maybe a meta data file?) to be imported in Blender to render a complete scene. I imagine having lights and the camera exported with the GLB. And Blender opens the file and starts rendering and saves the image. I tried to export exactely this, but the light sources and the camera are just empty/dead objects. 

I heard that glTF also supports light sources and cameras.

What is your opinion about this?

Best regards

Alex

**Comment by akutterxressential** [comment]
[SceneTest.zip](https://github.com/atteneder/glTFast/files/9567474/SceneTest.zip)

I attached the scene that I exported with glTFast (Unity 2021.2.16f1 and gltFast 4.8.5) I have to admit, I have no clue of Blender, but it is obvious that the camera is not recognized as camera and the lights not as lights.

**Comment by atteneder** [comment, marked as answer]
The upcoming version 5 of glTFast will support both cameras and lights. You can try it out today by using the `main` branch.

Cheers!
